### PR TITLE
feat(commands): add back support for other users' avatars

### PR DESCRIPTION
**Pull request by Ratismal**
The meme command used to have support for other member's avatars. This functionality was removed, and has since been requested. This PR's objective is to add back this functionality.

### CODE REVIEW

**Comment by luffykingpirate** [comment]
+1 pleaseee

**Comment by Stylite** [comment]
+1

File: src/main/kotlin/me/alexflipnote/kawaiibot/commands/Meme.kt [hunk 1]
```diff
@@ -4,28 +4,52 @@ import me.alexflipnote.kawaiibot.extensions.closing
...
     override fun run(ctx: CommandContext) {
+        val user : User
         val args = ctx.argString.split('|')
-        if (args.size != 2)
-            return ctx.send("You must pass exactly 2 arguments delimited by the pipe char |")
+        val topText: String
+        val bottomText: String
+
+        when {
+            args.size == 3 ->{
```
Comment (perryprog) [inline src/main/kotlin/me/alexflipnote/kawaiibot/commands/Meme.kt]: Literally unusable. (Space after `>` please)

**Comment by Ratismal** [comment]
@aurieh do you have any further comments or concerns on this pr other than"kinda hacky"? can we merge to fix the immediate issue, and redo at a later date to be less "hacky" when the framework allows?

**Pull request merged**